editor.planx.uk/src/pages/FlowEditor/components/Team/components/AddNewEditorModal.tsx

```diff
@@ -49,16 +51,19 @@ export const AddNewEditorModal = ({
       if (isUserAlreadyExistsError(err.message)) {
         setShowUserAlreadyExistsError(true);
       }
+      if (err.message === "Unable to create user") {
+        setShowErrorToast(true);
+      }
```

Comment:
Not sure if we'll ever reach this due to the error handling already in `src/lib/graphql.ts` but hopefully a good fallback